swo_aws_extension/crm_service_client/__init__.py

```diff
@@ -0,0 +1,3 @@
+from swo_aws_extension.crm_service_client.config import get_service_client
+
```

Comment:
Do you know any reason why do you have `crm_service_client` and `swo_crm_service_client` in the PR? What is the differences between them?